centreon/www/front_src/src/Authentication/WebSSO/index.test.tsx

```diff
@@ -25,51 +30,74 @@ import {
   labelInvalidIPAddress,
   labelInvalidRegex,
   labelLoginHeaderAttributeName,
-  labelMixed,
   labelPatternMatchLogin,
   labelPatternReplaceLogin,
   labelTrustedClientAddresses,
   labelWebSSOOnly
 } from './translatedLabels';
+import { retrievedWebSSOConfiguration } from './defaults';
 
 import WebSSOConfigurationForm from '.';
 
-const mockedAxios = axios as jest.Mocked<typeof axios>;
-
 jest.mock('../logos/providerPadlock.svg');
 
-const cancelTokenRequestParam = { cancelToken: {} };
-
-const cancelTokenPutParams = {
-  ...cancelTokenRequestParam,
-  headers: {
-    'Content-Type': 'application/x-www-form-urlencoded'
-  }
-};
-
 const renderWebSSOConfigurationForm = (): RenderResult =>
-  render(<WebSSOConfigurationForm />);
-
-const retrievedWebSSOConfiguration = {
-  blacklist_client_addresses: ['127.0.0.1'],
-  is_active: true,
-  is_forced: false,
-  login_header_attribute: '',
-  pattern_matching_login: null,
-  pattern_replace_login: null,
-  trusted_client_addresses: ['127.0.0.1']
-};
-
-describe('Web SSOconfiguration form', () => {
+  render(
+    <TestQueryProvider>
+      <WebSSOConfigurationForm />
+    </TestQueryProvider>
+  );
+
+describe('Web SSO configuration form', () => {
   beforeEach(() => {
-    mockedAxios.get.mockReset();
-    mockedAxios.get.mockResolvedValue({
+    resetMocks();
+    mockResponseOnce({
       data: retrievedWebSSOConfiguration
     });
+  });
+
+  it('saves the web SSO configuration when a field is modified and the "Save" button is clicked', async () => {
+    renderWebSSOConfigurationForm();
+
+    await waitFor(() => {
+      expect(getFetchCall(0)).toEqual(
+        authenticationProvidersEndpoint(Provider.WebSSO)
+      );
+    });
+
+    await waitFor(() => {
+      expect(screen.getByText(labelActivation)).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(
+        screen.getByLabelText(labelEnableWebSSOAuthentication)
+      ).toBeChecked();
+    });
+
+    userEvent.type(
+      screen.getByLabelText(labelLoginHeaderAttributeName),
+      'admin'
+    );
+    userEvent.tab();
+
+    await waitFor(() => {
+      expect(screen.getByText(labelSave)).not.toBeDisabled();
```

Comment:
if it can be disabled, it's surely accessible using `getByLabelText` or `getByRole`